UnitTests/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitorTest.cs

```diff
@@ -75,6 +75,23 @@ public void SimpleExpression_IsEvaluatable ()
     }
 
     [Test]
+    public void ExpressionTypeEnum_HasNoHoles ()
+    {
+      // We made a perf-optimization in RMLNQ-107 by avoiding calling Enum.IsDefined.
+      // This test verifies our assumption that the ExpressionType enum does not have holes.
+
+      var expected = 0;
+
+      foreach (var value in Enum.GetValues(typeof(ExpressionType)))
+      {
+        if (expected++ != (int)value)
```

Comment:
Please add a whitespace before between cast and value. Also, please add whitespaces in line 85 and 89.